src/gt4py/cartesian/gtc/debug/debug_codegen.py

```diff
@@ -247,7 +247,10 @@ def visit_FieldAccess(self, field_access: FieldAccess, **_) -> str:
             data_index_access = ",".join(
                 [self.visit(data_index) for data_index in field_access.data_index]
             )
-            full_string = field_access.name + "[" + offset_str + "," + data_index_access + "]"
+            if offset_str == "":
+                full_string = field_access.name + f"[{data_index_access}]"
+            else:
+                full_string = field_access.name + "[" + offset_str + "," + data_index_access + "]"
```

Comment:
How about we are a bit more pythonic?
```suggestion
            index_str = f"{offset_str},{data_index_access}" if offset_str else data_index_access
            full_string = f"{field_access.name}[{index_str}]"
```